commands/filemanager/__tests__/upload.test.ts

```diff
@@ -0,0 +1,47 @@
+import yargs, { Argv } from 'yargs';
+import {
+  addGlobalOptions,
+  addConfigOptions,
+  addAccountOptions,
+  addUseEnvironmentOptions,
+} from '../../../lib/commonOpts';
+
+jest.mock('yargs');
+jest.mock('../../../lib/commonOpts');
+
+// Import this last so mocks apply
```

Comment:
Nit: I see we have been adding this comment to the tests, but jest should be hoisting all of the jest.mock function calls to before the imports, so the actual order is irrelevant. 

https://jestjs.io/docs/manual-mocks#:~:text=For%20this%20reason%2C%20Jest%20will%20automatically%20hoist%20jest.mock%20calls%20to%20the%20top%20of%20the%20module%20(before%20any%20imports